locales/enUS.lua

```diff
@@ -229,6 +229,7 @@ app.L = {
 	["NESTED_QUEST_REQUIREMENTS"] = "Nested Quest Requirements";
 	["MAIN_LIST_REQUIRES_REFRESH"] = "[Open Main list to update progress]";
 	["DOES_NOT_CONTRIBUTE_TO_PROGRESS"] = "|cffe08207This group and its content do not contribute to the progress of this window!|r";
+	["ITEMS_NEEDED_TO_CRAFT"] = "Total items needed for crafting: ";
```

Comment:
You can add RU translation as well (I saw your screenshots are in Russian).

Reply:
I figured the wording is not the best and will likely change in the future, plus I hope to add similar calculation for other aspects so why bother for now, but in the end - yea, I will